ports/nrf/Makefile

```diff
@@ -41,6 +41,11 @@ QSTR_DEFS = qstrdefsport.h $(BUILD)/pins_qstr.h
 # MicroPython feature configurations
 MICROPY_ROM_TEXT_COMPRESSION ?= 1
 
+
+ifeq ($(DEBUG), 1)
+MICROPY_ROM_TEXT_COMPRESSION = 0
```

Comment:
Maybe move this down to line 125 where other DEBUG flags are set?

Reply:
Makes sense, changed.